### PR TITLE
Create True Black mode for amoled screens

### PR DESCRIPTION
**Pull request by paulolima18**
## Pull request (PR) checklist

Please check if your pull request fulfills the following requirements:

- [x] I've read the [Contribution Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/CONTRIBUTING.md) and my PR doesn't break the rules.
- [x] I've read and understand the [Developer Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/docs/Guidelines.md).
- [x] I confirm that I've run the code locally and everything works as expected.
- [x] My PR includes only the necessary changes to fix the issue (i.e., no unnecessary files or lines of code are changed).
- [x] 🎬 I've attached a **screen recording** of using the new code to the next paragraph (if applicable).

## Screen recording of your changes (if applicable):

https://github.com/user-attachments/assets/f0c55f7e-d628-4455-b4cb-23ab35df6fe1

## What's changed?

Describe with a few bullets **what's new:**
- I've added a true black mode for amoled screens

Before|After
---|---
Screenshot 2024-09-17 at 12 15 04 | Screenshot 2024-09-17 at 12 14 57

## Risk factors

**What may go wrong if we merge your PR?**
- People might not like the same icon for dark mode and true black mode
- ...

**In what cases won't your code work?**
- ...
- ...

## Does this PR close any GitHub issues? (do not delete)

- Closes #3538

## Troubleshooting GitHub Actions (CI) failures ❌
Pull request checks failing? Read our [CI Troubleshooting guide](https://github.com/Ivy-Apps/ivy-wallet/blob/main/docs/CI-Troubleshooting.md).

### CODE REVIEW

File: shared/ui/core/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -103,6 +103,7 @@
     <string name="settings">Settings</string>
     <string name="light_mode">Light mode</string>
     <string name="dark_mode">Dark mode</string>
+    <string name="amoled_mode">Black mode</string>
```
Comment (ILIYANGERMANOV) [inline shared/ui/core/src/main/res/values/strings.xml:106]: Rename to "Amoled Dark mode"

File: screen/home/src/main/java/com/ivy/home/HomeMoreMenu.kt [hunk 2]
```diff
@@ -445,21 +445,25 @@ private fun QuickAccess(
                 icon = when (theme) {
                     Theme.LIGHT -> R.drawable.home_more_menu_light_mode
                     Theme.DARK -> R.drawable.home_more_menu_dark_mode
+                    Theme.TRUE_BLACK -> R.drawable.home_more_menu_dark_mode
```
Comment (ILIYANGERMANOV) [inline screen/home/src/main/java/com/ivy/home/HomeMoreMenu.kt:448]: Can you add a different icon? Something from the material ones. Or the same material icon but with different fill. If doesn't look good, you can skip that

**Review by ILIYANGERMANOV — CHANGES_REQUESTED** [review]
LGTM! Left some comments. Apply them, fix the CI and we can merge. Nice work 💯

File: screen/home/src/main/java/com/ivy/home/HomeMoreMenu.kt (hunk 2 above)
Comment (paulolima18) [reply screen/home/src/main/java/com/ivy/home/HomeMoreMenu.kt:448]: What do you think of this way?
Screenshot 2024-09-17 at 16 47 52

Comment (ILIYANGERMANOV) [reply screen/home/src/main/java/com/ivy/home/HomeMoreMenu.kt:448]: LGTM! 👍

**Review by ILIYANGERMANOV — APPROVED** [review]

**Pull request merged**